pyop2/compilation.py

```diff
@@ -552,7 +546,7 @@ class AnonymousCompiler(Compiler):
 
 @mpi.collective
 def load(jitmodule, extension, fn_name, cppargs=(), ldargs=(),
-         argtypes=None, restype=None, comm=None):
+         argtypes=None, restype=None, comm=mpi.COMM_WORLD):
```

Comment:
Is `comm` getting passed correctly into this function by PyOP2?

Reply:
I will revert this change. Originally I was doing communication in this routine